reference/api/state_api.md

```diff
@@ -274,6 +275,79 @@ None.
 curl -X "DELETE" http://localhost:3500/v1.0/state/starwars/planet -H "ETag: xxxxxxx"
 ```
 
+## State transactions
+
+Persists the changes to the state store as a multi-item transaction.
+
+***Note that this operation is dependant on a using state store component that supports multi-item transactions.***
```

Comment:
Unless this is listed somewhere else, and I don't think this is, we need to put a list here of supported state stores.

Reply:
There is a kind of a list here, but it's in the context of actors.  Yeah there should be one for "state stores that support multi-item transactions" https://github.com/dapr/docs/blob/master/concepts/actors/actors_features.md#actor-state-management

Reply:
@youngbupark can you add the following list here as transactional state stores?

* Redis
* MongoDB
* PostgreSQL
* SQL Server
* Azure CosmosDB

Reply:
Added